CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ Unreleased
 ### Changed
 * Give webpack plugins a chance to manipulate devServer options.
 
+### Added
+* Fallback theme generation strategy for theme aggregation.
+* Theme visual regression support.
```

Comment:
These must have drifted out of the unreleased section in one of the master merges

Reply:
https://github.com/cerner/terra-toolkit/pull/324/commits/db42599dd9489c0198a0eb99e637cfe9d88b8ec8#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8